graphql/schemabuilder/pagination.go

```diff
@@ -302,31 +302,40 @@ func applyCursorsToAllEdges(allEdges []Edge, before *string, after *string) ([]E
 
 }
 
+func getEdges(key string, nodes []interface{}) (edges []Edge) {
```

Comment:
nit: `nodesToEdges`? `edgesFromPages`?

Reply:
should we put this `connectionContext` to grab the `key`?

Reply:
Happy to. Probably `nodesToEdges` 👍